manual/xsite-minikube/Makefile

```diff
@@ -126,8 +126,8 @@ create-cache:
 developer-password = $$(kubectl get secret \
     example-infinispan-generated-secret \
     -o jsonpath="{.data.identities\.yaml}" \
-    | base64 -D \
-    | yq -r .credentials[0].password)
+    | base64 -d \
```

Comment:
MacOS uses `-d` for debug. `base64 --decode` it's probably the portable syntax.

Reply:
Yes, --decode should be the best option here.